config/environments/production.rb

```diff
@@ -70,6 +71,12 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp-unencrypted.stanford.edu',
+    domain: 'stanford.edu'
+  }
```

Comment:
I don't think we need to set these as the servers are already configured correctly.  sul-requests and searchworks do not set these and they send emails.

Reply:
Removed and tested on stage